app/src/main/res/values/strings.xml

```diff
@@ -5,7 +5,6 @@
 <resources>
     <!-- Name of the application -->
     <string name="app_name">Fenix</string>
-
```

Comment:
nit: Are you intentionally removing the spacing here? I've added it to make it easier to read this file with the main headers being separated by a newline.

Reply:
good catch! Added them back